hardware_interface/src/joint_command_handle.cpp

```diff
@@ -21,38 +21,39 @@
 
 namespace hardware_interface
 {
-
 JointCommandHandle::JointCommandHandle()
-: name_(),
-  cmd_(nullptr)
-{}
+: name_(), cmd_(nullptr)
+{
+}
 
-JointCommandHandle::JointCommandHandle(
-  const std::string & name,
-  double * cmd)
+JointCommandHandle::JointCommandHandle(const std::string & name, double * cmd)
 : name_(name), cmd_(cmd)
 {
   THROW_ON_NULLPTR(cmd)
 }
 
-const std::string &
-JointCommandHandle::get_name() const
+const std::string & JointCommandHandle::get_name() const
 {
   return name_;
 }
 
-double
-JointCommandHandle::get_cmd() const
+double JointCommandHandle::get_cmd() const
 {
+  THROW_ON_NULLPTR(cmd_)
```

Comment:
I disagree on this one. We should check the pointer before de-referencing it

Reply:
Could you please elaborate on this one? This is checking the pointer before de-referencing it and throwing if null. These checks are required because there is a way to create handles with nullptr inside

Reply:
Must have been a visual thing. The change here looks good to me.